src/index.ts

```diff
@@ -131,3 +131,18 @@ export const requestAndConfirmAirdrop = async (
   );
   return connection.getBalance(publicKey, "confirmed");
 };
+
+export const airdrop = requestAndConfirmAirdrop;
```

Comment:
I strongly dislike this function existing. Why have a two names for the same function?

Reply:
Thanks. 

- Intention was to have a quick, memorable function to `airdrop()` with sensible defaults. 

- The name `requestAndConfirmAirdrop()` is what web3.js 2 uses.

What do you think? Should we use one of these (if so which one?), both (if the above changes anything), neither of these and a use better name, something else?

Reply:
I think we should have it, and use the "request" version. 
This will make it easier to remove this once web3js v2 is more common in the wild

Reply:
Good point, we don't want people using a name that will go away and if we want a short name we should go for it in web3. 

I've gotten rid of the short names and will file something separately for them upstream in web3.